compose/docker-compose.yml

```diff
@@ -82,16 +82,16 @@ services:
         ipv4_address: 192.17.5.15
 
 
-  # dashboard:
-  #   build:
-  #     context: ./..
-  #     dockerfile: Dockerfile
-  #   restart: always
-  #   ports:
-  #     - "80:80/tcp"
-  #   networks:
-  #     apisix-dashboard:
-  #       ipv4_address: 192.17.5.16
+  dashboard:
+    build:
+      context: ./..
+      dockerfile: Dockerfile
+    restart: always
+    ports:
+      - "80:80/tcp"
+    networks:
+      apisix-dashboard:
+        ipv4_address: 192.17.5.16
```

Comment:
too wired 🤔 is it a fixed IP address? @kv

Reply:
networks and IP is not necessary  in dashboard,you can remove the networks field